js/panel-manager.js

```diff
@@ -69,33 +75,62 @@ genotet.panelManager.togglePanel_ = function() {
 };
 
 /**
+ * Activate the side panel.
+ * @private
+ */
+genotet.panelManager.activatePanel_ = function(viewID) {
+  var tabID = 'panel-tab-' + viewID;
+  var tabContentID = 'panel-view-' + viewID;
+  $('.sideways li.active').removeClass('active');
+  $('#' + tabID).addClass('active');
+  $('.tab-content div.active').removeClass('active');
+  $('#' + tabContentID).addClass('active');
+},
+
+/**
  * Creates a panel with the given name.
  * @param {string} viewName Name of the view.
  */
 genotet.panelManager.addPanel = function(view) {
-  var viewID = view.name().replace(/\s/g, '');
+  var viewID = view.name().replace(/\s/g, '-');
```

Comment:
It is bad code to do such replace every time here. You may want to create a map that stores (originalName, renamedName).